pkg/scheduler/core/scheduling_queue.go

```diff
@@ -58,6 +62,7 @@ type SchedulingQueue interface {
 	AssignedPodUpdated(pod *v1.Pod)
 	WaitingPodsForNode(nodeName string) []*v1.Pod
 	WaitingPods() []*v1.Pod
+	Close()
```

Comment:
Could you please add comments to `Pop()` and `Close()` in this interface that explain the blocking behavior?

Reply:
sure.